src/main/java/org/sonarsource/scanner/maven/SonarQubeMojo.java

```diff
@@ -89,6 +90,7 @@ public void execute() throws MojoExecutionException, MojoFailureException {
       getLog().info("Delaying SonarQube Scanner to the end of multi-module project");
       return;
     }
+    System.out.println("hello");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS106))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.scanner.maven%3Asonar-maven-plugin&branch=feature%2Fjanos%2Fjunk&issues=AWFQYhIYsS4rYBZqE3Gh&open=AWFQYhIYsS4rYBZqE3Gh)

[ISSUE_KEY]: AWFQYhIYsS4rYBZqE3Gh